mvpa2/clfs/meta.py

```diff
@@ -1490,11 +1490,9 @@ def __init__(self, clf, centroids=None, distance_measure=None, **kwargs):
 
     def __repr__(self, prefixes=[]):
         if self.centroids is not None:
-            prefixes = prefixes + ['centroids=%r'
-                                   % self.centroids]
+            prefixes += ['centroids=%r' % self.centroids]
```

Comment:
This is not a mere beautification. It is changing into --inplace modification. Now try this on an object first which has centroids and then the one which didn't... I am afraid if would allow the bug. Copy prefixes first then at least